spec/system/candidate_interface/candidate_account_spec.rb

```diff
@@ -77,6 +83,14 @@ def and_i_submit_my_email_address(email = @email)
     click_on t('authentication.sign_up.button_continue')
   end
 
+  def and_i_submit_without_entering_an_email
+    click_on t('authentication.sign_up.button_continue')
+  end
+
+  def and_the_ts_and_cs_are_still_checked
+    expect(find_field(t('authentication.sign_up.accept_terms_checkbox')).value).to eq 'on'
```

Comment:
Capybara has a cool helper for this:

```suggestion
    expect(page).to have_checked_field t('authentication.sign_up.accept_terms_checkbox')
```

Reply:
Thanks, I've added that in now